### ISSUE TITLE
Setting `UpdatePresenceWhenStopped` is ignored and Presence shows impossible times

### ISSUE THREAD

**Issue by Svarr** [sll552/DiscordBee#81]
**Describe the bug**
Under certain conditions the Rich Presence is displayed in Discord even though `UpdatePresenceWhenStopped` is turned off and nothing is currently played back. The presence shows information about the last track that was played.

**To Reproduce**
I'm not entirely certain about which conditions can cause this bug to appear. Here is one way to reproduce it (that is, how this bug happened to me today):
1. Start MusicBee and play some music (don't run Discord yet, `UpdatePresenceWhenStopped` should be disabled)
2. Do some housework or waste time in whatever way you prefer, and let the playlist run out (maybe a certain time of being afk is required)
3. Eventually return to the PC (playback has already run out) and start Discord
4. Admire the Rich Presence showing MusicBee and the last song that has been played, while still counting up the playtime (way above the track's duration)

**Expected behavior**
Nothing shown in Discord except my "About Me" text and other stuff that is usually shown when clicking on my name in the userlist.

**Screenshots**
![discordbee](https://user-images.githubusercontent.com/4354904/130839945-c4afa288-678b-4b6b-999e-dfb2ec29d5c9.png)
(In case you didn't know, that song is far shorter than 3h. 😉 )

**Environment (please complete the following information):**
 - MusicBee Version: **3.4.7805**
 - Plugin Version: **1.4.2**

**Additional context**
I think the key is to start Discord after MusicBee, while playback is halted but something had been played previously in the same MusicBee session (so that `UpdateDiscordPresence` had been called with `PlayState.Playing` at least once).

It may be worth to look into what `DiscordRpcClient` does on initialisation when no Discord client is currently running and then later, when Discord is run.

**Comment by Svarr** [comment]
I found the cause for the exception and is indeed a bug in DiscordRPC (see [this issue](https://github.com/Lachee/discord-rpc-csharp/issues/155)).

But I forgot to make the crucial change of actually **not** sending the presence updates while not connected to Discord. This is fixed as of the latest commit to the branch `feature/lifecycle-fix` of my fork, linked in the post above.

I also tested whether the bug is still happening and it does seem to be fixed.
